projects/plugins/crm/src/entities/class-invoice.php

```diff
@@ -0,0 +1,310 @@
+<?php
+/**
+ * Invoice Entity.
+ *
+ * @package automattic/jetpack-crm
+ */
+
+namespace Automattic\Jetpack\CRM\Entities;
+
+/**
+ * Invoice class.
+ *
+ * @since $$next-version$$
+ */
+class Invoice {
+
+	/**
+	 * The DB ID of the object.
+	 *
+	 * @var int
+	 */
+	public $id = -1;
+
+	/**
+	 * The owner of the object.
+	 *
+	 * @var int
+	 */
+	public $owner = -1;
+
+	/**
+	 * The invoice meta.
+	 *
+	 * @var array
+	 */
+	public $meta = array();
+
+	/**
+	 * The invoice tags.
+	 *
+	 * @var array
+	 */
+	public $tags = array();
+
+	/**
+	 * The invoice files.
+	 *
+	 * @var array
+	 */
+	public $files = array();
+
+	/**
+	 * The invoice status.
+	 *
+	 * @var string
+	 */
+	public $status = 'Draft';
```

Comment:
I know it was in the reference draft-PR with the Contact entity. But we won't use default values, leaving it empty. (I'll do the same on Contact entity into another PR)

```suggestion
	public $status = '';
```